### PR TITLE
Completed post/put requests, added sorting

### PR DESCRIPTION
**Pull request by cmoralespv**
Implemented the helpfulness handler
Implemented the report handler
Added sorting logic to reviews list get request

### CODE REVIEW

File: src/api/routes/get.js [hunk 1]
```diff
@@ -9,18 +9,38 @@ router.get('/reviews/', (req, res) => {
...
+  console.log(typeof req.query.sort);
+
+  const sortList = (option) => {
+    console.log('option', option);
+    if (option === 'newest') {
+      sort = 'date DESC';
+    } else if (option === 'helpful') {
+      sort = 'helpfulness DESC';
+    } else {
+      sort = 'date DESC, helpfulness DESC';
```
Comment (jnbooth) [inline src/api/routes/get.js:25]: clever way to handle the typo issue

File: src/api/routes/post.js [hunk 2]
```diff
@@ -45,6 +50,30 @@ router.post('/reviews/', (req, res) => {
...
+});
+
+// ROUTE FOR: REPORT
+
+router.put('/reviews/:review_id/report', (req, res) => {
+  const review_id = req.params.review_id;
+
+  sql.query(
+    `UPDATE reviews SET reported = NOT reported
+    WHERE review_id = ${review_id};`)
```
Comment (jnbooth) [inline src/api/routes/post.js:72]: Currently a security flaw

**Review by jnbooth — APPROVED** [review]

**Comment by cmoralespv** [comment]
[Implement Mark Review as Helpful](https://trello.com/c/Kh3Ghuqo/28-implement-mark-review-as-helpful)

**Comment by cmoralespv** [comment]
[Added sorting to list reviews get request and refactored pagination to be like the heroku server](https://trello.com/c/IUj0xXgh/33-added-sorting-to-list-reviews-get-request-and-refactored-pagination-to-be-like-the-heroku-server)

**Pull request merged**